test/helpers/constants.js

```diff
@@ -1,7 +1,7 @@
 const MARKET_NAMES = {
   ZORA: 'ZORA',
   FOUNDATION: 'FOUNDATION',
-  NOUNS: 'NOUNS',
+  // NOUNS: 'NOUNS',
```

Comment:
Nouns tests are all broken, confirmed with anna that we can just remove this, but can be done in a separate pr